cql3/query_processor.cc

```diff
@@ -17,6 +17,7 @@
 #include "service/migration_manager.hh"
 #include "service/forward_service.hh"
 #include "service/raft/raft_group0_client.hh"
+#include "service/storage_service.hh"
```

Comment:
This patch needs a comment explaining why qp::remote needs to mess with storage service. Not that it can't be found out from next patches, but having it as text would help a lot

Reply:
Added.